esaco-app/src/test/java/it/infn/mw/esaco/test/IntrospectIntegrationTests.java

```diff
@@ -80,7 +86,9 @@ public void testUnreadableToken() throws Exception {
   @Test
   public void testIntrospectionWithInvalidToken() throws Exception {
 
-    when(tokenInfoService.isAccessTokenActive(Mockito.any())).thenReturn(false);
+    when(tokenIntrospectionService.introspectToken(Mockito.anyString()))
```

Comment:
I would use here introspectToken(VALID_JWT) for consistency with testIntrospectionWithValidToken